_data/tables/transform-high-performing-org.yml

```diff
@@ -0,0 +1,206 @@
+- table:
+  id: t001
+  name:
+    fr: Ensemble cohérent de mesures
+    en: Coherent set of actions
+  rows:
+  - row:
+    headers: true
+    cols:
+    - col:
+      data:
+        list: false
+        fr: Résultat
+        en: Outcome
+    - col:
+      data:
+        list: false
+        fr: Mesure
+        en: Action
+    - col:
+      data:
+        list: false
+        fr: Description
+        en: Description  
+    - col:
+      data:
+        list: false
+        fr: Parties prenantes
+        en: Stakeholders 
+  - row:
+    headers: false
+    cols:
+    - col:
+      data:
+        list: false
+        fr: Gouvernance
+        en: Governance
+    - col:
+      data:
+        list: false
+        fr: 
+        en: Establish communication plan and change management for continuous improvement in (and around) IITB
+    - col:
+      data:
+        list: false
+        fr: 
+        en: 
+    - col:
+      data:
+        list: true
+        items:
+        - item:
+            fr: <strong>BOP</strong>
+            en: <strong>BOP</strong>
+        - item:
+            fr: Stratégie TI
+            en: IT Strategy
+  - row:
+    headers: false
+    cols:
+    - col:
+      data:
+        list: false
+        fr: Gouvernance
+        en: Governance
+    - col:
+      data:
+        list: false
+        fr: 
+        en: Develop a self assessement tool based on capabilities of high performing organizations and PwC recommendations
+    - col:
+      data:
+        list: false
+        fr: 
+        en: 
+    - col:
+      data:
+        list: true
+        items:
+        - item:
+            fr: <strong>Stratégie TI</strong>
+            en: <strong>IT Strategy</strong>
+        - item:
+            fr: R&P
+            en: R&P
+  - row:
+    headers: false
+    cols:
+    - col:
+      data:
+        list: false
+        fr: Gouvernance
+        en: Governance
+    - col:
+      data:
+        list: false
+        fr: 
+        en: Promote ways for users to propose innovation and experimentation (Grassroots, PoC, ..)
```

Comment:
Bit open for interpretation, do we have an exhaustive list of "actionable" items?